core/site.go

```diff
@@ -789,19 +794,25 @@ func (site *Site) update(lp updater) {
 		flexiblePower = site.prioritizer.GetChargePowerFlexibility(lp)
 	}
 
-	var smartCostActive bool
-	if rate, err := site.plannerRate(); err == nil {
-		smartCostActive = site.smartCostActive(lp, rate)
-	} else {
-		site.log.WARN.Println("smartCostActive:", err)
+	// battery mode handling
+	rates, err := site.plannerRates()
+	if err != nil {
+		site.log.WARN.Println("planner:", err)
+	}
+
+	rate, err := rates.Current(time.Now())
+	if rates != nil && err != nil {
+		site.log.WARN.Println("planner:", err)
 	}
 
-	var smartCostNextStart time.Time
-	if !smartCostActive {
```

Comment:
Die Akzeptanztests schlagen hier fehl, weil diese Bedingung beim Umbau in den lp rausgeflogen ist. Wenn Smart Cost aktiv ist, soll kein "nächster Zeitpunkt" zurückgegeben werden.

Reply:
Siehe https://github.com/evcc-io/evcc/actions/runs/10031691085/job/27722454946?pr=14855#step:10:2363
Da wird eine Uhrzeit angezeigt, aber keine erwartet.